colors/material.vim

```diff
@@ -220,6 +220,13 @@ call s:SetHighlight('Underlined', s:blue, '', '')
 call s:SetHighlight('Error', s:bg, s:red, '')
 call s:SetHighlight('Todo', s:orange, s:bg, 'italic')
 
+" diff misnames
```

Comment:
Can you explain what this comment means? Not sure I understand.

Reply:
well, in the theme you correctly applied colors to 
`DiffAdd`
`DiffAdded`
`DiffChange`
etc.

however in git diff, these groups are called differently for some reason.

I can delete this comment if it's confusing.

I'm also not sure about `diffFile`, `diffNewFile` and `diffLine` color.

Reply:
Are the highlight names below coming from a specific plugin? I do think the comment is confusing - it would be nice to add a comment describing why these are needed (ideally, where they are coming from).

Reply:
this comes from `git diff`, the file type is `diff`. Here is a similar issue in `nord` theme repo:
https://github.com/arcticicestudio/nord-vim/issues/66

Reply:
I copied the comment from nord theme